keyboards/vortex/m0110/info.json

```diff
@@ -0,0 +1,393 @@
+                { "label": "1", "matrix": [1, 7], "x": 1, "y": 0 },
+                { "label": "2", "matrix": [2, 7], "x": 2, "y": 0 },
+                { "label": "3", "matrix": [3, 7], "x": 3, "y": 0 },
+                { "label": "4", "matrix": [4, 7], "x": 4, "y": 0 },
+                { "label": "5", "matrix": [4, 6], "x": 5, "y": 0 },
+                { "label": "6", "matrix": [5, 6], "x": 6, "y": 0 },
+                { "label": "7", "matrix": [5, 7], "x": 7, "y": 0 },
+                { "label": "8", "matrix": [6, 7], "x": 8, "y": 0 },
+                { "label": "9", "matrix": [7, 7], "x": 9, "y": 0 },
+                { "label": "0", "matrix": [8, 7], "x": 10, "y": 0 },
+                { "label": "-", "matrix": [8, 6], "x": 11, "y": 0 },
+                { "label": "=", "matrix": [6, 6], "x": 12, "y": 0 },
+                { "label": "\\", "matrix": [10, 2], "x": 13, "y": 0 },
+                { "label": "del", "matrix": [6, 5], "x": 14, "y": 0 },
+                
+                { "label": "Tab", "matrix": [1, 1], "w": 1.5, "x": 0, "y": 1 },
+                { "label": "Q", "matrix": [1, 0], "x": 1.5, "y": 1 },
+                { "label": "W", "matrix": [2, 0], "x": 2.5, "y": 1 },
+                { "label": "E", "matrix": [3, 0], "x": 3.5, "y": 1 },
+                { "label": "R", "matrix": [4, 0], "x": 4.5, "y": 1 },
+                { "label": "T", "matrix": [4, 1], "x": 5.5, "y": 1 },
+                { "label": "Y", "matrix": [5, 1], "x": 6.5, "y": 1 },
+                { "label": "U", "matrix": [5, 0], "x": 7.5, "y": 1 },
+                { "label": "I", "matrix": [6, 0], "x": 8.5, "y": 1 },
+                { "label": "O", "matrix": [7, 0], "x": 9.5, "y": 1 },
+                { "label": "P", "matrix": [8, 0], "x": 10.5, "y": 1 },
+                { "label": "[", "matrix": [8, 1], "x": 11.5, "y": 1 },
+                { "label": "]", "matrix": [6, 1], "x": 12.5, "y": 1 },
+                { "label": "Enter", "matrix": [10, 4], "w": 1.25, "h": 2, "x": 13.75, "y": 1 },
+                
+                { "label": "Caps Lock", "matrix": [2, 1], "w": 1.75, "x": 0, "y": 2 },
+                { "label": "A", "matrix": [1, 2], "x": 1.75, "y": 2 },
+                { "label": "S", "matrix": [2, 2], "x": 2.75, "y": 2 },
+                { "label": "D", "matrix": [3, 2], "x": 3.75, "y": 2 },
+                { "label": "F", "matrix": [4, 2], "x": 4.75, "y": 2 },
+                { "label": "G", "matrix": [4, 3], "x": 5.75, "y": 2 },
+                { "label": "H", "matrix": [5, 3], "x": 6.75, "y": 2 },
+                { "label": "J", "matrix": [5, 2], "x": 7.75, "y": 2 },
+                { "label": "K", "matrix": [6, 2], "x": 8.75, "y": 2 },
+                { "label": "L", "matrix": [7, 2], "x": 9.75, "y": 2 },
+                { "label": ";", "matrix": [8, 2], "x": 10.75, "y": 2 },
+                { "label": "'", "matrix": [8, 3], "x": 11.75, "y": 2 },
+                { "label": "#", "matrix": [0, 2], "x": 12.75, "y": 2 },
```

Comment:
```suggestion
                
                { "label": "Caps Lock", "matrix": [2, 1], "w": 1.75, "x": 0, "y": 2 },
                { "label": "A", "matrix": [1, 2], "x": 1.75, "y": 2 },
                { "label": "S", "matrix": [2, 2], "x": 2.75, "y": 2 },
                { "label": "D", "matrix": [3, 2], "x": 3.75, "y": 2 },
                { "label": "F", "matrix": [4, 2], "x": 4.75, "y": 2 },
                { "label": "G", "matrix": [4, 3], "x": 5.75, "y": 2 },
                { "label": "H", "matrix": [5, 3], "x": 6.75, "y": 2 },
                { "label": "J", "matrix": [5, 2], "x": 7.75, "y": 2 },
                { "label": "K", "matrix": [6, 2], "x": 8.75, "y": 2 },
                { "label": "L", "matrix": [7, 2], "x": 9.75, "y": 2 },
                { "label": ";", "matrix": [8, 2], "x": 10.75, "y": 2 },
                { "label": "'", "matrix": [8, 3], "x": 11.75, "y": 2 },
                { "label": "#", "matrix": [0, 2], "x": 12.75, "y": 2 },
                { "label": "Enter", "matrix": [10, 4], "w": 1.25, "h": 2, "x": 13.75, "y": 1 },
```

Reply:
@noroadsleft If I move "Enter" key to 3rd row then my layout becoming wrong...so, I am keeping enter key in 2nd row.

Reply:
I have to agree with @noroadsleft with this.

If Enter is moved to 3rd row in `layouts`, then it'll also need to be moved to 3rd row in keymap(s); thankfully is really not a difficult change/ask.